spec/cfn_nag_integration/cfn_nag_rds_instance_spec.rb

```diff
@@ -30,7 +30,60 @@
       expect(actual_aggregate_results).to eq expected_aggregate_results
     end
   end
-end
 
+  context 'one RDS instance with default credentials and no-echo is true' do
+    it 'flags a violation' do
+      template_name = 'json/rds_instance/rds_instance_no_echo_with_default_password.json'
+
+      expected_aggregate_results = [
+        {
+          filename: test_template_path(template_name),
+          file_results: {
+            failure_count: 2,
+            violations: [
+              Violation.new(id: 'F23',
```

Comment:
one minor problem i have here is that the violations are not ordered. tested with a  sample template which gives me a similar result. https://gist.github.com/xysr89/41c6d18be6b913fbcff9f2b29dae0d45

not really an issue but its worth to bring it up